src/js/util/immutable-utils.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+function pathToArr(pathStr) {
+  return pathStr.split('.');
+}
+
+function getIn(structure, pathStr) {
```

Comment:
I would add comments here to define that the structure is an immutable object  (it is right? am I crazy?)